.travis.yml

```diff
@@ -0,0 +1,33 @@
+# This is a common .travis.yml for generating library release zip files for
+# CircuitPython library releases using circuitpython-build-tools.
+# See https://github.com/adafruit/circuitpython-build-tools for detailed setup
+# instructions.
+
+dist: xenial
+language: python
+python:
+    - "3.6"
+
+cache:
+    pip: true
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_TOKEN
+  file_glob: true
+  file: $TRAVIS_BUILD_DIR/bundles/*
+  skip_cleanup: true
+  overwrite: true
+  on:
+    tags: true
+
+install:
+  - pip install -r requirements.txt
+  - pip install circuitpython-build-tools Sphinx sphinx-rtd-theme
+  - pip install --force-reinstall pylint==1.9.2
+
+script:
+  - pylint adafruit_motorkit.py
+  - ([[ ! -d "examples" ]] || pylint --disable=missing-docstring,invalid-name,bad-whitespace examples/*.py)
+  - circuitpython-build-bundles --filename_prefix adafruit-circuitpython-motorkit --library_location .
+  - cd docs && sphinx-build -E -W -b html . _build/html
```

Comment:
Need a `&& cd ..` here?

Reply:
Not yet. I'm going to do the PyPI setup in a second PR. Easier to troubleshoot issues.